tests/test_sam_resource.py

```diff
@@ -246,6 +245,16 @@ def test_preload_sam_sites(sites):
     assert np.allclose(truth, test)
 
 
+def test_bad_site_req():
+    """Test that the preload_SAM method raises an error on a bad site request
+    """
+    sites = [0, 10, 500]
```

Comment:
why are these bad sites? it is bc there arent 500 sites in the h5 here?

Reply:
correct, will add comment